apps/arweave/src/ar_nonce_limiter_server.erl

```diff
@@ -0,0 +1,94 @@
+-module(ar_nonce_limiter_server).
```

Comment:
Is this module primarily refactoring? Or does the PR also add new functionality? (Just wondering how closely I should review the logic for changes in behavior)